agentops/llm_tracker.py

```diff
@@ -407,9 +467,17 @@ def generator():
             self.llm_event.completion_tokens = response.meta.tokens.output_tokens
             self.llm_event.model = kwargs.get("model", "command-r-plus")
 
-            self.client.record(self.llm_event)
+            if session is not None:
```

Comment:
can just do if session: to stay consistent

Reply:
added helper function